test/test_saemref.py

```diff
@@ -80,5 +80,5 @@ def test_saemref_running(Process, Service, Socket, Command, supervisor_service_n
 
 
 def test_saemref_sync_source_cronjob(Command):
-    jobs = Command.check_output("crontab -u saemref -l").splitlines()
-    assert '* */1 * * * CW_MODE=user cubicweb-ctl source-sync --loglevel error saemref' in jobs
+    jobs = Command.check_output("crontab -u saemref -l")
+    assert 'cubicweb-ctl source-sync --loglevel error saemref' in jobs
```

Comment:
Isn't the virtualenv prefix missing from `cubicweb-ctl` executable? I'd expect a full path like `/home/saem/venv/bin/cubicweb-ctl` here.